src/Sql/dbo/Stored Procedures/Collection_DeleteByIds.sql

```diff
@@ -0,0 +1,54 @@
+CREATE PROCEDURE [dbo].[Collection_DeleteByIds]
+    @Ids AS [dbo].[GuidIdArray] READONLY
+AS
+BEGIN
+    SET NOCOUNT ON
+
+    DECLARE @OrgIds AS [dbo].[GuidIdArray]
+
+    INSERT INTO @OrgIds (Id)
+    SELECT
+        [OrganizationId]
+    FROM
+        [dbo].[Collection]
+    WHERE
+        [Id] in (SELECT [Id] FROM @Ids)
+    GROUP BY
+        [OrganizationId]
+
+    DECLARE @BatchSize INT = 100
+	
+    -- Delete Collection Groups
+    WHILE @BatchSize > 0
```

Comment:
Similar to the question in the EF repository, do we need to delete the `CollectionGroups` if they're automatically deleted when the `Collection` is removed?

Reply:
Answered above